ConorMFinn/CO600-SeriousCyberSec
# Add Animation to Sherry's Movement

**Description of work.**
Sherry is now animated!
Added animations for the 4 movement directions
Also added single frame idle animations for when they stop. 
Uses Unity's built in animator system, switching between Idle and Walking states. 
Makes use of a Blend Tree to decide on the direction based on the X and Y inputs. 
Also did some refactoring of the movement script.

**To test:**
1. Move sherry about the scene.
2. Check they move properly as before.
3. Check animations look good and make sense (speed, direction, etc)
4. Check stopping switches to an idle animation. 

- Code Review: `SherryMovement.cs` has been refactored and changed. 

Fixes #36
